packages/flutter/lib/src/material/chip.dart

```diff
@@ -2440,24 +2460,20 @@ class _RenderChip extends RenderBox {
   }
 
   @override
-  bool hitTest(BoxHitTestResult result, { Offset position }) {
-    if (!size.contains(position))
+  bool hitTest(BoxHitTestResult result, {Offset position}) {
```

Comment:
Put the spaces back on the named parameter here.

```suggestion
  bool hitTest(BoxHitTestResult result, { Offset position }) {
```

(This is a flutter style pattern)

Reply:
Fixed.